src/util/ownerDocument.js

```diff
@@ -0,0 +1,5 @@
+function ownerDocument(node) {
```

Comment:
I'm not quite sure I understand what this function does. Would you mind adding a comment explaining its purpose?